src/Common/Entity/Cell.php

```diff
@@ -37,7 +36,7 @@ final public function getStyle(): Style
         return $this->style;
     }
 
-    final public static function fromValue(mixed $value, ?Style $style = null): self
+    final public static function fromValue(null|bool|string|int|float|DateTimeInterface|DateInterval $value, ?Style $style = null): self
```

Comment:
I do not consider this a BC break since the library just broke if any other type where used.